### PR TITLE
Added a short tutorial to resolve conflicts

### PR DESCRIPTION
**Pull request by danirueda**
I've added a short tutorial that teachs how to fix git conflicts resolving issue #45

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+```
+
+remote: Counting objects: 7, done.
+remote: Compressing objects: 100% (4/4), done.
+remote: Total 4 (delta 3), reused 0 (delta 0)
+Unpacking objects: 100% (4/4), done.
+De github.com:oslugr/curso-git
+   afee5ab..bf454ef  master     -> origin/master
+Auto-merging texto/mas-usos.md
+CONFLIC(contenido): merge conflict/mas-usos.md
```
Comment (fjlopez) [inline description.md]: Fix typo in CONFLIC

File: description.md [hunk 2]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+
+A conflict is the modification of the same line or lines by two or more users. A conflict appears when we try to merge branches.
+It's important to know how many branches we have in our repository. With `git log` we can see the commit's history, with
+`git log --oneline` we get the same result but in a short version. But, with `git log --graph --oneline` we get a graphic
+vision of our commits and branches. Also it exists the command `git show-branch -a` that shows all the repository branches.
+
+When a conflict happens, the console result is like this
+
+```
+
```
Comment (fjlopez) [inline description.md]: Remove extra line

File: description.md [hunk 3]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+We can see, that git tells to us which files have a conflict. If we open one of them, there will be something like this
+
+```
+
+<<<<<<< HEAD
+    ## Resolving git conflicts
+    =======
```
Comment (fjlopez) [inline description.md]: remove extra whitespace

File: description.md [hunk 4]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
 
 As we can see our project follows Mavens directory layout. There are some directories in the layout that aren't present in our project structure but this is because our project source doesn't use anything from those directories since they are generated at building time.
 
+#HOW TO RESOLVE CONFLICTS IN GIT
+The first question that we've to do ourselves is, what is a conflict?
```
Comment (fjlopez) [inline description.md]: Technical writing should not use wordiness.

File: description.md [hunk 5]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+
+A conflict is the modification of the same line or lines by two or more users. A conflict appears when we try to merge branches.
+It's important to know how many branches we have in our repository. With `git log` we can see the commit's history, with
+`git log --oneline` we get the same result but in a short version. But, with `git log --graph --oneline` we get a graphic
+vision of our commits and branches. Also it exists the command `git show-branch -a` that shows all the repository branches.
+
+When a conflict happens, the console result is like this
+
```
Comment (fjlopez) [inline description.md]: replace result by output

File: description.md [hunk 6]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+```
+
+remote: Counting objects: 7, done.
+remote: Compressing objects: 100% (4/4), done.
+remote: Total 4 (delta 3), reused 0 (delta 0)
+Unpacking objects: 100% (4/4), done.
+De github.com:oslugr/curso-git
+   afee5ab..bf454ef  master     -> origin/master
```
Comment (fjlopez) [inline description.md]: De? This look spanish

File: description.md [hunk 7]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+Auto-merging texto/mas-usos.md
+CONFLIC(contenido): merge conflict/mas-usos.md
+Automatic merge failed; fix conflicts and then commit the result.
+
```
Comment (fjlopez) [inline description.md]: Remove this empty line

File: description.md [hunk 8]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+```
+
+We can see, that git tells to us which files have a conflict. If we open one of them, there will be something like this
+
+```
```
Comment (fjlopez) [inline description.md]: Add a hint for [code and syntax highlighting](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)

File: description.md [hunk 9]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+
```
Comment (fjlopez) [inline description.md]: Remove this empty line

File: description.md [hunk 10]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+<<<<<<< HEAD
+    ## Resolving git conflicts
+    =======
+    ## Let's see how we can reolve git conflicts.
+    >>>>>>> bf454eff1b2ea242ea0570389bc75c1ade6b7fa0
+
```
Comment (fjlopez) [inline description.md]: Remove this empty line.

File: description.md [hunk 11]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+```
+
+What we have is between the first line and '=======' and, what are between '=======' and the last line is the code
```
Comment (fjlopez) [inline description.md]: Use backticks for code. In addition, review the sentence. I think that something is missing.

File: description.md [hunk 12]
```diff
@@ -139,6 +139,52 @@ In our case, our project has the next structure:
...
+of the other branch. What we've to do is delete the confict lines (<<<,===,>>>) and choose which code we want. We must do the same
+for all the conflict files. Finally, we can merge branches without problems.
+
+There are a lot of graphic tools to resolve conflicts, but this is the simply way to do it.
+
+This part resolves issue #45.
```
Comment (fjlopez) [inline description.md]: Add a link to the issue

File: description.md (hunk 6 above)
Comment (danirueda) [inline description.md]: I copied the examples from a spanish tutorial.

File: description.md [hunk 13]
```diff
@@ -161,49 +161,158 @@ In our case, our project has the next structure:
...
 We can see, that git tells to us which files have a conflict. If we open one of them, there will be something like this
 
 ```
-
-<<<<<<< HEAD
+<<<<<<<HEAD
     ## Resolving git conflicts
     =======
     ## Let's see how we can reolve git conflicts.
     >>>>>>> bf454eff1b2ea242ea0570389bc75c1ade6b7fa0
```
Comment (fjlopez) [inline description.md]: This is extra whitespace.

File: description.md [hunk 14]
```diff
@@ -161,49 +161,158 @@ In our case, our project has the next structure:
...
 ```
 
 We can see, that git tells to us which files have a conflict. If we open one of them, there will be something like this
 
 ```
-
-<<<<<<< HEAD
+<<<<<<<HEAD
     ## Resolving git conflicts
     =======
```
Comment (fjlopez) [inline description.md]: This is extra whitespace.

File: description.md [hunk 15]
```diff
@@ -161,49 +161,158 @@ In our case, our project has the next structure:
...
-What we have is between the first line and '=======' and, what are between '=======' and the last line is the code
+What we have between the first line and `=======` and, what are between `=======` and the last line is the code
 of the other branch. What we've to do is delete the confict lines (<<<,===,>>>) and choose which code we want. We must do the same
 for all the conflict files. Finally, we can merge branches without problems.
 
 There are a lot of graphic tools to resolve conflicts, but this is the simply way to do it.
 
-This part resolves issue #45.
+This part resolves issue #[45](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/issues/45).
+
```
Comment (fjlopez) [inline description.md]: This block tells me that you should do a sync prior to the next push to GitHub. This code is already on the master.

File: description.md (hunk 13 above)
Comment (danirueda) [inline description.md]: Which whitespace? I don't see it.

Comment (fjlopez) [inline description.md]: Do you see the space between the left side of the code and ##, ======  or >>>>>>>>?

File: description.md (hunk 6 above)
Comment (fjlopez) [inline description.md]: I think that you should follow a hands-on approach. In addition, doing a copy&paste from a source without citing the source is a bad practise. Always give the attribution to the author unless he states that this is unnecessary.

**Comment by danirueda** [comment]
@fjlopez Is there something wrong? I deleted de whitespaces and the issue is linked.

**Pull request merged**